src/components/TouchpadController.js

```diff
@@ -0,0 +1,63 @@
+import React, { useContext, useEffect, useState } from 'react';
+
+import { makeStyles } from '@material-ui/core/styles';
+import Fab from '@material-ui/core/Fab';
+import KeyboardArrowDown from '@material-ui/icons/KeyboardArrowDown';
+import KeyboardArrowUp from '@material-ui/icons/KeyboardArrowUp';
+import KeyboardArrowLeft from '@material-ui/icons/KeyboardArrowLeft';
+import KeyboardArrowRight from '@material-ui/icons/KeyboardArrowRight';
+
+import { Context as ControllerContext } from '../context/Controller';
+
+const margin = 23;
+const useStyles = makeStyles({
+    root: {
+        textAlign: 'center'
+    },
+    leftButton: {
+        marginRight: margin,
+    },
+    rightButton: {
+        marginLeft: margin,
+    },
+  });
+
+export default function TouchpadController() {
+    const classes = useStyles();
```

Comment:
What do you think about doing destructuring?

Reply:
that's not how it's documented...